command/sub8_alarm/sub8_alarm/alarm_helpers.py

```diff
@@ -0,0 +1,80 @@
+import rospy
+from sub8_msgs.msg import Alarm
+from std_msgs.msg import Header
+import json
+
+class AlarmBroadcaster(object):
+    def __init__(self):
+        '''Alarm Broadcaster'''
+        # Notify the world when our alarm happens
+        self.alarm_pub = rospy.Publisher('/alarm', Alarm, latch=True, queue_size=10)
+
+        # Notify the world that our alarm has spawned
+        # For now, unused - future: Use for system reports
+        # self.alarm_spawn_pub = rospy.Publisher('/alarm_spawn', Alarm, latch=True, queue_size=1)
+
+        # Get ROS name
+        self.node_name = rospy.get_name()
+
+        # Alarms associated with this broadcaster
+        # In the future, there will be some background bookkeeping
+        self.alarms = []
+
+    def add_alarm(self, name, action_required=False, severity=5, problem_description=None, parameters=None):
```

Comment:
Update wiki with severity levels; the wiki currently only has descriptions for levels 0-2